test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -105,7 +105,7 @@ private void TrainRegression(string trainDataPath, string testDataPath, string m
                 // once so adding a caching step before it is not helpful.
                 .AppendCacheCheckpoint(mlContext)
                 // Add the SDCA regression trainer.
-                .Append(mlContext.Regression.Trainers.StochasticDualCoordinateAscent(labelColumnName: "Target", featureColumnName: "FeatureVector"));
+                .Append(mlContext.Regression.Trainers.Sdca(labelColumnName: "Target", featureColumnName: "FeatureVector"));
```

Comment:
when we update the cookbook samples files, it usually means the cookbook needs to be updated.

Reply:
thanks for pointing this out!

---
In reply to: [264879257](https://github.com/dotnet/machinelearning/pull/2903#discussion_r264879257) [](ancestors = 264879257)